### ISSUE TITLE
Dune won’t run JavaScript tests “no rule found for .js_dom.inline-tests/out.js”

### ISSUE THREAD

**Issue by TyOverby** [ocaml/dune#2916]
```
$ cat dune
(library
 (name js_dom)
 (libraries core_kernel)
 (js_of_ocaml (javascript_files ./out.js ./runtime.js))
 (inline_tests 
   (modes js)
   (deps ./out.js ./runtime.js))
 (preprocess
  (pps ppx_jane)))

$ dune build @@runtest
Entering directory '/home/tyoverby/workspace/ocaml/vdm'
Error: No rule found for jsdom/.js_dom.inline-tests/out.js
Error: No rule found for jsdom/.js_dom.inline-tests/runtime.js

$ cat ../dune-project
(lang dune 1.11)
```

**Comment by nojb** [comment]
Which version of dune are you using ?

**Comment by TyOverby** [comment]
> Which version of dune are you using ?

1.11.4

**Comment by rgrinberg** [comment]
Is this fixed in 2.7 for you?

**Comment by hhugo** [comment]
I suspect this can be closed
